src/notation/view/widgets/editstyle.ui

```diff
@@ -14729,45 +14747,10 @@ followed by dashes</string>
   <tabstop>resetAutoplaceVerticalAlignRange</tabstop>
   <tabstop>minVerticalDistance</tabstop>
   <tabstop>resetMinVerticalDistance</tabstop>
-  <tabstop>scrollArea_page</tabstop>
```

Comment:
You'll need to reset the tab order on this page using the "Edit Tab Order" button in the design view